[Docbug][Addnote][4125] (#4182)
MicrosoftDocs/msteams-docs: msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md

## Use the shared meeting stage

Shared meeting stage helps meeting participants interact with and collaborate on app content in real time. For example, users can focus their meeting on editing a document, brainstorming with a whiteboard, or reviewing a dashboard.

Apps shared to the meeting stage occupy the same space as a shared screen. The stage reorients for all meeting participants.

>[!NOTE] 

> >Apps shared to the meeting stage from desktop is accessible only for the mobile meeting users. 

Reviewer: Please rephrase:
```suggestion
Currently, if an app is shared to stage on desktop it will show for the users on mobile meeting only.
```

#### Meeting stage (without side panel)

> When the side panel isn't open, the meeting stage is 994x678 pixels by default and can be a minimum 792x382 pixel.

Reviewer: Please check for consistency, 
```suggestion
When the side panel isn't open, the meeting stage is at 994x678 pixels by default, and can be a minimum 792x382 pixels.
```

:::image type="content" source="~/assets/images/apps-in-meetings/meeting-stage-no-side-panel.png" alt-text="Image showing shared meeting stage responsiveness with the side panel closed." border="false":::

#### Meeting stage (with side panel)

> When the side panel is open, the meeting stage is 918x540 pixels by default and can be a minimum 472x382 pixel.

Reviewer: Please check for singular/ plural, the word pixel is used in plural, please make it plurals.
